client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -1,197 +1,163 @@
+								: 'https://jetpack.com/support/infinite-scroll/'
+						}
+						privacyLink="https://jetpack.com/support/infinite-scroll/#privacy"
+					/>
+					<FormSettingExplanation>
 						{ translate(
-							'Your site has a "footer" widget enabled so buttons will always be used. {{link}}Customize your site{{/link}}',
-							{
-								components: {
-									link: <a href={ customizeUrl } />,
-								},
-							}
+							'Create a smooth, uninterrupted reading experience by loading more content as visitors scroll to the bottom of your archive pages.'
 						) }
 					</FormSettingExplanation>
-				) }
-			</FormFieldset>
-		);
-	}
-
-	renderJetpackInfiniteScrollSettings() {
-		const { translate } = this.props;
-
-		return (
-			<FormFieldset>
-				<SupportInfo
-					text={ translate(
-						'Loads the next posts automatically when the reader approaches the bottom of the page.'
-					) }
-					link="https://jetpack.com/support/infinite-scroll/"
-				/>
-				<FormLegend>{ translate( 'Infinite Scroll' ) }</FormLegend>
-				<p>
-					{ translate(
-						'Create a smooth, uninterrupted reading experience by loading more content as visitors scroll to the bottom of your archive pages.'
-					) }
-				</p>
-				{ this.renderRadio(
-					'infinite_scroll',
-					'default',
-					translate( 'Load more posts using the default theme behavior' )
-				) }
-				{ this.renderRadio(
-					'infinite_scroll',
-					'button',
-					translate( 'Load more posts in page with a button' )
-				) }
-				{ this.renderRadio(
-					'infinite_scroll',
-					'scroll',
-					translate( 'Load more posts as the reader scrolls down' )
-				) }
-			</FormFieldset>
-		);
-	}
-
-	renderCustomCSSSettings() {
-		const { selectedSiteId, translate } = this.props;
-		const formPending = this.isFormPending();
-
-		return (
-			<FormFieldset>
-				<SupportInfo
-					text={ translate(
-						"Adds options for CSS preprocessor use, disabling the theme's CSS, or custom image width."
-					) }
-					link="https://jetpack.com/support/custom-css/"
-				/>
-
-				<JetpackModuleToggle
-					siteId={ selectedSiteId }
-					moduleSlug="custom-css"
-					label={ translate( 'Enhance CSS customization panel' ) }
-					disabled={ formPending }
-				/>
-			</FormFieldset>
-		);
-	}
-
-	render() {
-		const { siteIsJetpack, translate } = this.props;
-
-		/* eslint-disable wpcalypso/jsx-classname-namespace */
-		return (
-			<div>
-				<SettingsSectionHeader title={ translate( 'Theme enhancements' ) } />
-
-				<Card className="theme-enhancements__card site-settings">
-					{ siteIsJetpack ? (
-						<Fragment>
-							{ this.renderJetpackInfiniteScrollSettings() }
-							<hr />
-							{ this.renderCustomCSSSettings() }
-						</Fragment>
-					) : (
-						this.renderSimpleSiteInfiniteScrollSettings()
+					<RadioOptions />
+					<hr />
+					<SupportInfo
+						text={ translate(
+							"Adds names for CSS preprocessor use, disabling the theme's CSS, or custom image width."
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Adds options for CSS preprocessor use, disabling the theme's CSS, or custom image width.' )` *ES Score: 10*